syntax = "proto3";

package hello_world.v1;

service HelloWorldAPI {
  rpc SayHello(SayHelloRequest) returns (SayHelloResponse);
}

message SayHelloRequest {
  string name = 1;
}

message SayHelloResponse {
  string response = 1;
}